pkg/doc/verifiable/common.go

```diff
@@ -121,3 +121,51 @@ func stringSlice(values []interface{}) ([]string, error) {
 
 	return strings, nil
 }
+
+// decodeType decodes raw type(s).
+//
+// type can be defined as a single string value or array of strings.
+func decodeType(t interface{}) ([]string, error) {
+	switch rType := t.(type) {
+	case string:
+		return []string{rType}, nil
+	case []interface{}:
+		types, err := stringSlice(rType)
+		if err != nil {
+			return nil, fmt.Errorf("vc types: %w", err)
+		}
+
+		return types, nil
+	default:
+		return nil, errors.New("credential type of unknown type")
+	}
+}
+
+// decodeContext decodes raw context(s).
+//
+// context can be defined as a single string value or array;
+// at the second case, the array can be a mix of string and object types
+// (objects can express context information); object context are
+// defined at the tail of the array.
+func decodeContext(c interface{}) ([]string, []interface{}, error) {
```

Comment:
`decodContext()` was used for `Credential`, now is used by `Presentation` as well.